tests/rptest/services/redpanda.py

```diff
@@ -1586,21 +1591,24 @@ def start_redpanda(self, node):
 
         node.account.ssh(cmd)
 
-    def all_up(self):
-        def check_node(node):
-            pid = self.redpanda_pid(node)
-            if not pid:
-                self.logger.warn(f"No redpanda PIDs found on {node.name}")
-                return False
+    def check_node(self, node):
+        pid = self.redpanda_pid(node)
+        if not pid:
+            self.logger.warn(f"No redpanda PIDs found on {node.name}")
+            return False
 
-            if not node.account.exists(f"/proc/{pid}"):
-                self.logger.warn(f"PID {pid} (node {node.name}) dead")
-                return False
+        if not node.account.exists(f"/proc/{pid}"):
+            self.logger.warn(f"PID {pid} (node {node.name}) dead")
+            return False
 
-            # fall through
-            return True
+        # fall through
+        return True
 
-        return all(self._for_nodes(self._started, check_node, parallel=True))
+    def all_up(self):
+        return all(
+            self._for_nodes(self._started,
```

Comment:
This function is not available in redpanda.py any more. Probably a good idea to rebase it.